R/ppmify.R

```diff
@@ -302,6 +305,36 @@ grid <- function (area, density) {
 
 }
 
+bias <- function (area, bias_points, offset.raster=FALSE){
+  
+  # calculate weights for integration points when using pseudo-observations as integration points
+  # weight = total study area / number of integration/quadrature points
+  
+  if (offset.raster){ # if offset raster is also provided, calculate sum of raster values
+    
+    a <- cellStats(area, sum, na.rm=TRUE)  
```

Comment:
On reflection, I think that 'biasRaster' was a misleading name for this method. I agree that a raster of sampling effort should be incorporated as you suggested. 

At present 'biasRaster' allows me to incorporate bias points and a population raster for incorporating the proportion susceptible to infection i.e. background offset = log (proportion susceptible) + log(total_population) - log(number of bias points)...I might need to think about how to make this generalizable